notsobot/notsobot.py

```diff
@@ -1592,12 +1682,16 @@ async def invert(self, ctx, urls: ImageFinder = None):
 
             def invert_img(b):
                 with Image.open(b).convert("RGB") as img:
-                    img = ImageOps.invert(img)
+                    image = ImageOps.invert(img)
+                    img.close()
                 with BytesIO() as final:
-                    img.save(final, "png")
+                    image.save(final, "png")
                     file_size = final.tell()
                     final.seek(0)
-                    file = discord.File(final, filename="flop.png")
+                    filename = self.random_filename(True, "png")
+                    file = discord.File(final, filename=filename)
+                    image.close()
+                    final.close()
```

Comment:
This is unnecessary since `final` was created by the context manager which closes it automatically.